### PR TITLE
Complete Rewrite

### PR DESCRIPTION
**Pull request by malayamanas**
#### Descriptions:
Baracudda RBL Connector

#### Fix:
None

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "name": "baracuddaRBL",
+    "version": "1.0.0",
+    "label": "Baracudda - DNS RBL",
```
Comment (cs-prerna-joshi) [inline info.json]:
```suggestion
    "label": "Baracudda Reputation Block List",
```

File: info.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "description": "IP Domain reputation lookup in Baracudda",
+    "publisher": "Fortinet",
+    "icon_small_name": "barracuda_32x32.png",
+    "icon_large_name": "barracuda_80X80.png",
+    "category": "TIP",
```
Comment (cs-prerna-joshi) [inline info.json]:
```suggestion
    "category": "Threat Intelligence",
```

File: info.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                "required": true,
+                "visible": true,
+                "editable": true
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "IP_Domain_rbllookup",
+            "category": "TIP",
```
Comment (cs-prerna-joshi) [inline info.json]:
```suggestion
            "category": "investigation",
```

File: info.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            "annotation": "IP_Domain_rbllookup",
```
Comment (cs-prerna-joshi) [inline info.json]:
```suggestion
            "annotation": "ip_domain_rbllookup",
```

File: info.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            "output_schema": {},
```
Comment (cs-prerna-joshi) [inline info.json]: add output schema.

File: info.json [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+            "title": "Lookup IP from Baracudda RBL",
```
Comment (cs-prerna-joshi) [inline info.json]:
```suggestion
            "title": "Get IP Reputation",
```

**Review by cs-prerna-joshi — COMMENTED** [review]
Please move all these file under folder named "barracuda-reputation-block-list"

File: info.json (hunk 5 above)
Comment (malayamanas) [reply info.json]: Added output schema.

Comment (malayamanas) [reply info.json]: Added

**Comment by malayamanas** [comment]
Please review @cs-prerna-joshi . I have added the changes as per your review input.

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**